File: src/main/java/aquarius0715/commandcountplugin/CommandCountPlugin.java
package aquarius0715.commandcountplugin;

import org.bukkit.plugin.java.JavaPlugin;
import org.bukkit.scoreboard.Objective;
import org.bukkit.scoreboard.Scoreboard;
import org.bukkit.scoreboard.ScoreboardManager;

import java.util.ArrayList;
import java.util.List;
import java.util.Objects;

public final class CommandCountPlugin extends JavaPlugin {

    List<PlayerData> playerData = new ArrayList<PlayerData>(10);

    int allScore;

    boolean pluginStats = true;
    boolean gameStats = false;
    boolean joinOnTheWay = true;

    ScoreboardManager scoreboardManager;
    Scoreboard scoreboard;
    Objective objective;

    ScoreBoard scoreBoard = new ScoreBoard(this);
    ScoreBoardData scoreBoardData = new ScoreBoardData(this);
    SQLInsert sqlInsert = new SQLInsert(this);
    MySQLManager MySQLManager = new MySQLManager(this, "CommandCountPlugin");
    SQLSelect sqlSelect = new SQLSelect(this);
    SQLUpdate sqlUpdate = new SQLUpdate(this);
    DateFormant dateFormant = new DateFormant(this);
    Timer timer = new Timer(this);

    String StartDate;

    int time = 0;

    @Override
    public void onEnable() {
        this.saveDefaultConfig();
        Objects.requireNonNull(getCommand("cmdcount")).setExecutor(new Commands(this));
        this.getServer().getPluginManager().registerEvents(new Events(this), this);

        MySQLManager = new MySQLManager(this, "CommandCountPlugin");


    }

    static class PlayerData {

        String playerName;
        int score;

        public PlayerData(String playerName, int score) {
            this.playerName = playerName;
            this.score = score;
        }
    }

    @Override
    public void onDisable() {
        // Plugin shutdown logic
    }
}

File: src/main/java/aquarius0715/commandcountplugin/CommandCountPlugin.kt
package aquarius0715.commandcountplugin

import org.bukkit.plugin.java.JavaPlugin
import org.bukkit.scoreboard.Objective
import org.bukkit.scoreboard.Scoreboard
import org.bukkit.scoreboard.ScoreboardManager
import java.util.*

class CommandCountPlugin : JavaPlugin() {
    var playerData: MutableList<PlayerData?> = ArrayList(10)
    var allScore = 0
    var pluginStats = true
    var gameStats = false
    var scoreboardManager: ScoreboardManager? = null
    var scoreboard: Scoreboard? = null
    var objective: Objective? = null
    var scoreBoard = ScoreBoard(this)
    var scoreBoardData = ScoreBoardData(this)
    var sqlInsert = SQLInsert(this)
    var MySQLManager = MySQLManager(this, "CommandCount")
    var sqlSelect = SQLSelect(this)
    var sqlUpdate = SQLUpdate(this)
    var dateFormant = DateFormant(this)
    var timer = Timer(this)
    var StartDate: String? = null
    var time = 0
    var addScoreOp = 0
    var days = 0
    var hours = 0
    var minutes = 0
    var seconds = 0
    var prefix = "§l[§d§lM§f§la§a§ln§f§l10§b§lレイド§f§l]"
    override fun onEnable() {
        saveDefaultConfig()
        Objects.requireNonNull(getCommand("cmdcount"))!!.setExecutor(Commands(this))
        server.pluginManager.registerEvents(Events(this), this)
        MySQLManager = MySQLManager(this, "CommandCountPlugin")
    }

    class PlayerData(var playerName: String, var score: Int)

    override fun onDisable() {
        // Plugin shutdown logic
    }
}